examples/eventloop/c_eventloop.c

```diff
@@ -14,6 +14,11 @@
 #include <poll.h>
 
 #include "duktape.h"
+#include "c_eventloop.h"
+
+#ifndef DUKTAPE_EVENTLOOP_DEBUG
```

Comment:
By convention `#if !defined(...)` is used throughout the code base.

Reply:
Will change that and resubmit. Should I somehow cancel this current pull request?

Reply:
No need, just squash the commits (e.g. git rebase -i master).